lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -91,17 +96,24 @@ def acceptable_pattern?(arg)
             arg.match?(/[ >,+]/)
           end
 
-          def specific_matcher_option?(arg, matcher)
+          def specific_matcher_option?(node, arg, matcher)
             # If `button[foo-bar_baz=foo][bar][baz]`:
             # extract ["foo-bar_baz", "bar", "baz"]
             attributes = arg.scan(/\[(.*?)(?:=.*?)?\]/).flatten
             return true if attributes.empty?
+            return false if matcher == 'link' &&
+              !replaceable_to_have_link?(node, attributes)
```

Comment:
IMO this would be easier to read if it is
```suggestion
            return false unless replaceable_matcher?(node, matcher, attributes)
```
and this will serve as a placeholder for other matchers, even though right now it would have implementation only for links
```
  def replaceable_matcher?(node, matcher, attributes)
    case matcher
    when 'link' then replaceable_to_have_link?(node, attributes)
    else true
    end
```

Reply:
It certainly looks good for increased readability. Thank you very much.